### PR TITLE
feat(combo-button): add menuOffset and menuOffsetFlip props for inner OverflowMenu

### LINKED ISSUE #720: Adding a menuOffset property to ComboButton to add custom styling
Adding a menuOffset property to ComboButton to add custom styling

 Details

Just like OverflowMenu , if a menuOffset property exists for ComboButton, we can apply some custom styling to the menu options. 
Without this, since our comboButton is rendered at the bottom most of the screen, the menu options are getting hidden.

### PR DESCRIPTION
**Pull request by jendowns**
## Affected issues

- Resolves #720

## Proposed changes

- add `menuOffset` prop with default, apply to inner `OverflowMenu`
- add `menuOffsetFlip` prop with default, apply to inner `OverflowMenu`

### CODE REVIEW

File: src/components/ComboButton/ComboButton.js [hunk 1]
```diff
@@ -169,6 +170,28 @@ ComboButton.propTypes = {
   /** @type {string} Overflow menu direction. */
   direction: PropTypes.oneOf([TooltipDirection.TOP, TooltipDirection.BOTTOM]),
 
+  /**
+   * The adjustment in position applied to the floating menu.
+   */
+  menuOffset: PropTypes.oneOfType([
+    PropTypes.shape({
+      top: PropTypes.number,
+      left: PropTypes.number,
```
Comment (diego-codes) [inline src/components/ComboButton/ComboButton.js:179]: Could these values also be strings, like `'auto'`?

Comment (jendowns) [reply src/components/ComboButton/ComboButton.js:179]: Good point! Will update that 👍

**Pull request merged**